### DISCUSSION TITLE
Hot reload doesn't seem to work from the demo project

### DISCUSSION THREAD

**Discussion by mustafa-hanif** [nuejs/nue#15] [Q&A]
I used option 1 while using bun https://nuejs.org/docs/nuejs/getting-started.html#option-1-create-nue and when I am changing any content in content.data and saving, page doesn't reload automatically

**Comment by mustafa-hanif** [comment]
Update: Doesn't work with nodeJS either.

**Reply by Axellarator** [reply]
There is no Hot Reload feature  available soon. Work in progress.

**Comment by tipiirai** [comment]
Hot reloading is part of the upcoming Nuekit project, which is the final piece on the whole toolchain. See: https://nuejs.org/tools/

I can see it being handy on the nuejs project itself, but it takes some effort to make it work reliably. Not sure I want to start maintaining that piece just yet.
